src/enums/entityTypeInfo.ts

```diff
@@ -0,0 +1,25 @@
+import { CorpTypeCd } from '@bcrs-shared-components/corp-type-module'
+
+// Define a map of entity types to titles and info strings
+export const entityTypeInfo = {
```

Comment:
Not sure if it's worth moving these to the already existing resource files?

Reply:
good catch! I'll update.